src/components/network-map-viewer/network/network-map.jsx

```diff
@@ -0,0 +1,565 @@
+                    }
+                },
+            })
+        );
+    }
+
+    const initialViewState = {
+        longitude: props.initialPosition[0],
+        latitude: props.initialPosition[1],
+        zoom: props.initialZoom,
+        maxZoom: 12,
+        pitch: 0,
+        bearing: 0,
+    };
+
+    const renderOverlay = () => (
+        <LoaderWithOverlay
+            color="inherit"
+            loaderSize={70}
+            isFixed={false}
+            loadingMessageText={'loadingGeoData'}
+        />
+    );
+
+    return (
+        <>
+            <DeckGL
+                onViewStateChange={onViewStateChange}
+                ref={(ref) => {
+                    // save a reference to the Deck instance to be able to center in onAfterRender
+                    setDeck(ref && ref.deck);
+                }}
+                onClick={(info, event) => {
+                    onClickHandler(info, event, props.mapEquipments);
+                }}
+                onAfterRender={onAfterRender}
+                layers={layers}
+                initialViewState={initialViewState}
+                controller={{ doubleClickZoom: false }}
+                ContextProvider={MapContext.Provider}
+                getCursor={cursorHandler}
+                pickingRadius={5}
+            >
+                {props.displayOverlayLoader && renderOverlay()}
+                {mapManualRefresh && reloadMapNeeded && currentNodeBuilt && (
+                    <Box sx={styles.mapManualRefreshBackdrop}>
+                        <Button
+                            onClick={props.onReloadMapClick}
+                            aria-label="reload"
+                            color="inherit"
+                            size="large"
+                        >
+                            <ReplayIcon />
+                            <FormattedMessage id="ManuallyRefreshGeoData" />
+                        </Button>
+                    </Box>
+                )}
+
+                {mToken && (
+                    <StaticMap
+                        mapStyle={theme.mapboxStyle}
+                        preventStyleDiffing={true}
+                        mapboxApiAccessToken={mToken}
+                    >
+                        {showTooltip && renderTooltip()}
+                    </StaticMap>
+                )}
+                <NavigationControl style={{ right: 10, top: 10, zIndex: 1 }} />
+            </DeckGL>
+        </>
+    );
+};
+
+NetworkMap.defaultProps = {
+    mapEquipments: null,
+    updatedLines: [],
+    geoData: null,
+    filteredNominalVoltages: null,
+    labelsZoomThreshold: 9,
+    arrowsZoomThreshold: 7,
+    tooltipZoomThreshold: 7,
+    initialZoom: 5,
+    initialPosition: [0, 0],
+    lineFullPath: true,
+    lineParallelPath: true,
+    lineFlowMode: LineFlowMode.FEEDERS,
+    lineFlowHidden: true,
+    lineFlowColorMode: LineFlowColorMode.NOMINAL_VOLTAGE,
+    lineFlowAlertThreshold: 100,
+    loadFlowStatus: RunningStatus.IDLE,
```

Comment:
We could get rid of the loadflow status notion. This props could be a boolean named areTransitsValid. And we change opacity of this labels depending on this props